src/cli/server/server.rs

```diff
@@ -33,6 +33,21 @@ impl Server {
     /// Starts the server in the current Runtime
     pub async fn start(self) -> Result<()> {
         let blueprint = Blueprint::try_from(&self.config_module).map_err(CLIError::from)?;
+        
+        let mut pathsMap = &HashMap::new();
+        for type_ in self.config_module.config.types.values() {
+            for field in type_.fields.values() {
+                if let Some(http) = &field.http {
+                    pathsMap.insert(http.path, http.on_request);
+                }
+            }
+        }
+        let server_config = Arc::new(ServerConfig::new(
+            blueprint.clone(),
+            self.config_module.extensions.endpoints,
+            pathsMap,
```

Comment:
@coderabbitai
Make necessary syntax changes in the codebase so that it reaches and available inside on_request function of RequestFilter in file src/cli/javascript/request_filter.rs
Here https://github.com/webbdays/tailcall-fork/blob/b10014f2caa276fd85df363bc067963bf1e38f04/src/cli/javascript/request_filter.rs#L37
Commit those changes to this branch.
Make sure syntax is correct.
Come with changes and show me.

Reply:
@coderabbitai
proceed for above comment

Reply:
@coderabbitai
Just give commitable suggestions.

Reply:
oK. We need to correct all files connecting from that fnc in  server.rs to on_request func in request_filter.rs file.